examples/advanced-ops-outbrain/01-Download-Convert.ipynb

```diff
@@ -52,8 +52,11 @@
     "import os\n",
     "\n",
     "import cupy\n",
-    "import cudf\n",
-    "import rmm"
+    "import rmm\n",
```

Comment:
we want to require rmm even though we dont have cudf? What uses rmm other than dask?

Reply:
That is a good point. We are using dask even in CPU Version with no cudf installed. I assumed we would need rmm becuase of that, but maybe it is not needed. @rjzamora may be able to confirm.

Reply:
@rjzamora could you confirm when possible please.

Reply:
My understanding is that RMM (i.e. "RAPIDS Memory Manager") is only useful/available for cuda-enabled systems (at least for now).  RMM can be used with both cudf and cupy. Therefore, we shouldn't tie an RMM dependency to Dask, but to these other GPU-focused libraries.

Reply:
If rmm is tied to GPU only and then it should not be installed on systems that do not have GPUs. So it needs to be abstracted out IMO. @albert17